prof.py

```diff
@@ -90,33 +90,34 @@ def get_bin_vals(self, counts_img, bkg_img,
                 j, k = pixel[0], pixel[1]
                 if exp_img_data[i][j, k] != 0:
                     exp_val = exp_img_data[i][j, k]
+                    exp_val_bkg = exp_img_data[i][j, k] * \
+                                  bkg_img_hdr[i]['EXPOSURE'] / \
+                                  counts_img_hdr[i]['EXPOSURE']
                     npix += 1
                 else:
                     continue
                 raw_cts += counts_img_data[i][j, k]
                 net_cts += counts_img_data[i][j, k] - \
                     bkg_img_data[i][j, k] * bkg_corr_i
```

Comment:
Could pull this out of the loop... just `net_cts = raw_cts - bkg_cts` at the end.